modules/user/lib/user-sync-wallet-balance.service.ts

```diff
@@ -77,17 +77,11 @@ export class UserSyncWalletBalanceService {
             endBlock = Math.min(endBlock, beetsBarBlock.number);
         }
 
-        const pools = await prisma.prismaPool.findMany({
-            select: { id: true, address: true },
-            where: { dynamicData: { totalSharesNum: { gt: 0.000000000001 } }, chain: this.chain },
```

Comment:
Is the thought that this might cause the missing balances?

The reason to filter for totalShares is that otherwise there are a shitton of pools